src/adapters/scene.js

```diff
@@ -130,31 +130,35 @@ Scene.prototype.initMapBox = async function(locationHash) {
     // we have to delay click event resolution to make time for possible double click events,
     // which are thrown *after* two separate click events are thrown
     this.clickDelayHandler = null;
-    this.mb.on('click', e => {
-      if (e.originalEvent.cancelBubble) {
-        return;
-      }
-      // cancel the previous click handler if it's still pending
-      clearTimeout(this.clickDelayHandler);
-      // if this is a real mouse double-click, we can simply return here
-      if (e.originalEvent.detail >= 2) {
-        return;
-      }
-      const pois = this.mb.queryRenderedFeatures(e.point, { layers: interactiveLayers });
-      // when clicking on a POI, just trigger the action without delay,
-      // as a subsequent double click isn't a problem
-      if (pois[0]) {
-        this.clickOnMap(e.lngLat, pois[0]);
-        return;
-      }
-      this.clickDelayHandler = setTimeout(() => {
-        // for touch UX we have to make sure a double tap zoom hasn't been made in the meantime
-        if (Date.now() - this.lastDoubleTapTimeStamp < this.DOUBLE_TAP_DELAY_MS) {
+
+    // iframe: disable clicks on map
```

Comment:
early return to avoid touching existing code parts ?

Reply:
an early return would cancel all the following code including mouse/finger interactions (that's OK) but also the final lines:
```
    window.execOnMapLoaded = f => f();
    fire('map_loaded');
```
I think we can't cut these.